src/govuk/components/textarea/textarea.yaml

```
@@ -41,20 +41,20 @@ params:
 - name: formGroup
   type: object
   required: false
-  description: Options for the form-group wrapper
+  description: Options for the form-group wrapper.
   params:
   - name: classes
     type: string
     required: false
-    description: Classes to add to the form group (for example to show error state for the whole group)
+    description: Classes to add to the form group (for example to show error state for the whole group).
 - name: classes
   type: string
   required: false
   description: Classes to add to the textarea.
 - name: autocomplete
   type: string
   required: false
-  description: Attribute to [identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html), for instance "postal-code" or "username". See [autofill](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) for full list of attributes that can be used.
+  description: Attribute to [identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html), for example 'postal-code' or 'username'. See [autofill](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) for full list of attributes that can be used.
```

Again wondering if these should be considered code…

```suggestion
  description: Attribute to [identify input purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html), for example `postal-code` or `username`. See [autofill](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) for full list of attributes that can be used.
```

---

Yeah, great shout. Will note for future reference. Thanks!